config/settings.yml

```diff
@@ -559,7 +560,8 @@ defaults: &defaults
         settings: &settings_plan3
           prefix: 'blueprint'
           agent: *agent_blueprint
-          template: <%= base64_template('blueprint') %>
+          template: <%= base64_template('blueprint', 'manifest') %>
```

Comment:
why confuse with this 'type' parameter. Type then must be enumerated. Why not pass the full file name like 'blueprint-manifest' or 'blueprint-dashboard'. Also this is to be included conditionally, as not all services will have dashboard template. Need to raise PR even on service-fabrik-product component with sample for dashboard.

Reply:
PR on sfpc is raised. There also i have added an example for blueprint dashboard.